app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,13 @@
+<resources
+        xmlns:tools="http://schemas.android.com/tools"
+        tools:ignore="MissingTranslation">
+        <string name="app_name">Kaffee</string>
+        <string name="keep_display_on">Bildschirm eingeschaltet lassen</string>
+        <string name="tap_to_turn_off">Tippen zum Ausschalten</string>
+        <string name="hint_1">Diese App stellt in den Schnelleinstellungen (Das Menü, wo man W-LAN und Bluetooth ein- und ausschalten kann) eine Kachel zur Verfügung, welche es erlaubt, den Bildschirm eingeschaltet zu lassen, ohne die Geräteeinstellungen zu ändern</string>
```

Comment:
```suggestion
        <string name="hint_1">Diese App stellt in den Schnelleinstellungen (der Ort, an dem die Schalter für z.B. WLAN und Bluetooth sind) eine Kachel zur Verfügung, welche es erlaubt, den Bildschirm eingeschaltet zu lassen, ohne die Geräteeinstellungen zu ändern</string>
```

Reply:
That definitely sounds nicer. I was unsure about that myself but couldn't think of anything better